integrated_channels/canvas/client.py

```diff
@@ -46,6 +48,8 @@ def __init__(self, enterprise_configuration):
         super(CanvasAPIClient, self).__init__(enterprise_configuration)
         self.config = apps.get_app_config('canvas')
         self.session = None
+        # Wondering if we want to keep track of a timeout on the access token to recall "get auth token"
+        self.session_time_out = None
```

Comment:
not sure why this is here but perhaps we can investigate this separately? I think you meant what happens if the access token expires? I guess we will have to handle that somehow by refetching a new token, it would be cool for us to create a ticket for that if you are inclined.